Week3/homework/6-step3.js

```diff
@@ -1,6 +1,20 @@
-'use strict';
+"use strict";
 
 const arr2d = [[1, 2], [3, 4], [5, 6]];
 const arr3d = [[[1, 2], [3, 4]], [[5, 6], [7, 8]]];
 
-// add your solution here, or add a comment on how you would tackle this problem
+for (let i = 0; i < arr3d.length; i++) {
+  for (let j = 0; j < arr3d[i].length; j++) {
+    for (let k = 0; k < arr3d[i][j].length; k++) {
+
+    }
+  }
+}
+
+// With for of
+
+for (let item of arr3d) {
```

Comment:
This `for` loop prints just the elements of the outer array. These elements happen to be elements themselves. I expected each individual value of the array to be printed out by your `for` loops.